ilc/client.js

```diff
@@ -5,38 +5,56 @@ import Router from './client/ClientRouter';
 import setupErrorHandlers from './client/errorHandler/setupErrorHandlers';
 import {fragmentErrorHandlerFactory, crashIlc} from './client/errorHandler/fragmentErrorHandlerFactory';
 import isActiveFactory from './client/isActiveFactory';
-import initIlcConfig from './client/initIlcConfig';
+import getIlcConfig from './client/ilcConfig';
 import initIlcState from './client/initIlcState';
 import setupPerformanceMonitoring from './client/performance';
 import selectSlotsToRegister from './client/selectSlotsToRegister';
-import {getSlotElement} from './client/utils';
+import {getSlotElement, getAppSpaCallbacks, prependSpaCallback} from './client/utils';
 import AsyncBootUp from './client/AsyncBootUp';
+import IlcAppSdk from 'ilc-sdk/app';
+import I18n from './client/i18n';
+import {makeAppId} from './common/utils';
 
 const System = window.System;
 if (System === undefined) {
     crashIlc();
     throw new Error('ILC: can\'t find SystemJS on a page, crashing everything');
 }
 
-const registryConf = initIlcConfig();
+const registryConf = getIlcConfig();
 const state = initIlcState();
-const router = new Router(registryConf, state, singleSpa);
+
+const appErrorHandlerFactory = (appName, slotName) => {
+    return fragmentErrorHandlerFactory(registryConf, router.getCurrentRoute, appName, slotName);
+};
+
+const i18n = registryConf.settings.i18n.enabled
+    ? new I18n(registryConf.settings.i18n, singleSpa, appErrorHandlerFactory)
+    : null;
+const router = new Router(registryConf, state, i18n ? i18n.unlocalizeUrl : undefined, singleSpa);
```

Comment:
it's not critical, but will be better to put `undefined` arguments as last argument or as optional inside object. and in this case we won't set `undefiend` here.
https://github.com/namecheap/ilc/pull/230/files#diff-318e72ee3c278cb96c1bb2130002e8a9bcdebd2042222be2a5284e589a1bb75eR204